lockfiles/requirements-build.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.13
+# This file is autogenerated by pip-compile with Python 3.12
```

Comment:
You may want to check and rebuild your local dev environment since it looks like your poetry may be using the older Python version.

Reply:
That being said, I believe 3.12 technically should still work, but I think the rest of us have moved to 3.13, and we started explicitly changing support for 3.13 a couple months ago (https://github.com/quipucords/quipucords/pull/2769).

Reply:
one could say this is a good moment to install uv and then...

```
uv python install 3.13 && uv python pin 3.13
uv tool install 'poetry<2'
poetry env remove --all && poetry env use $(uv python find) && poetry install
```

😉

Reply:
Definetely!

Reply:
@bruno-fs so fast 😮

Reply:
>so fast

![image](https://github.com/user-attachments/assets/b8cd1481-afad-490f-9002-67de7b50e3b5)